apps/vanilla/src/pages/blackJack/models/player.ts

```diff
@@ -0,0 +1,51 @@
+import { Subscriber } from '../type/subscriber';
+
+import { Publisher } from './publisher';
+const WIN_SCORE = 21;
+
+/** Player with publisher and subscriber func. */
+export class Player extends Publisher<number> implements Subscriber<number> {
+	/** All dice results for one player. */
+	public readonly diceResults: number[] = [];
+
+	/**
+	 * Push result dice roll in general array.
+	 * @param dice - Number.
+		*/
+	public result(dice: number): void {
+		this.diceResults.push(dice);
+	}
+
+	/** Get sum of all Dices. */
+	public getSumDices(): number {
+		return this.diceResults.reduce((acc, dice) => acc + dice, 0);
+	}
+
+	/** Check win status in general array. */
+	public getWinStatus(): boolean {
+		if (this.diceResults.reduce((acc, dice) => acc + dice, 0) >= WIN_SCORE) {
+			return true;
+		}
+
+		return false;
```

Comment:
```suggestion
		return this.diceResults.reduce((acc, dice) => acc + dice, 0) >= WIN_SCORE;
```

Reply:
By the way, you can improve this code further:
```suggestion
		return this.getSumDices() >= WIN_SCORE;
```